src/gamepad/gamepad-mappings-manager.vala

```diff
@@ -22,6 +22,13 @@ private class Games.GamepadMappingsManager : Object {
 		catch (Error e) {
 			warning ("GamepadMappingsManager: Can't find gamecontrollerdb.txt: %s", e.message);
 		}
+		try {
+			var dir = Application.get_config_dir ();
+			add_from_file (@"$dir/gamecontrollerdb.txt");
```

Comment:
It's probably cleaner to have "gamescontrollerdb.txt" as a private constant, and to use it here and un the error message: if for any reason we want to change it in the future it will change everywhere and it avoids "magic values" to be hidden into the code.